tests/tests_app/cli/test_cli.py

```diff
@@ -52,7 +52,7 @@ def test_commands(command):
 
 def test_main_lightning_cli_no_arguments():
     """Validate the Lightning CLI without args."""
-    res = os.popen("python -m lightning").read()
+    res = os.popen("lightning").read()
```

Comment:
I think we shall test both

Reply:
-m will fail when PACKAGE_NAME=app

Reply:
can we write an issue about it? this seems we shall notify users...
cc: @manskx

Reply:
It's not an issue but expected.

`python -m foo` bypasses the `console_scripts` entrypoint and calls the module `foo` directly. With `PACKAGE_NAME=app` we don't install the `lightning` module so the failure is working as expected